stable/dask/templates/dask-scheduler-deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "dask.scheduler-fullname" . }}
+  labels:
+    app: {{ template "dask.name" . }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Release.Name }}-{{ .Values.scheduler.component }}"
+spec:
+  replicas: {{ .Values.scheduler.replicas }}
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: {{ template "dask.name" . }}
+        release: {{ .Release.Name | quote }}
+        component: "{{ .Release.Name }}-{{ .Values.scheduler.component }}"
+    spec:
+      containers:
+        - name: {{ template "dask.scheduler-fullname" . }}
+          image: 
+            repository: {{ .Values.scheduler.image | quote }}
+            tag: {{ .Values.scheduler.imageTag | quote }}
+            pullPolicy: IfNotPresent
+          args: ["dask-scheduler", "--port", "{{ .Values.scheduler.servicePort }}", "--bokeh-port", "{{ .Values.webUI.containerPort }}"]
+          ports:
+            - containerPort: {{ .Values.scheduler.containerPort }}
```

Comment:
Any reason why you make container ports configurable. Changing them may break things. I'd also give them a name and refer to them by name in the services.